src/transformers/utils/doc.py

```diff
@@ -204,41 +223,53 @@ def _prepare_output_docstrings(output_type, config_class, min_indent=None):
     >>> import torch
     >>> from transformers import {processor_class}, {model_class}
 
-    >>> torch.manual_seed(0)  # doctest: +IGNORE_RESULT
-
     >>> tokenizer = {processor_class}.from_pretrained("{checkpoint}")
-    >>> model = {model_class}.from_pretrained("{checkpoint}", num_labels=2)
+    >>> model = {model_class}.from_pretrained("{checkpoint}")
```

Comment:
This will break all classification examples apart from RoBERTa that just fixed, no?

It's also the most common mistake people do with sequence classification labels, so we should show a value here (whatever value it is).